### PR TITLE
Configured Github Actions

### PR DESCRIPTION
**Pull request by nnzJd**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request'
+    permissions:
+      contents: read
+      packages: write
+    needs:
+      - publish
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```
Comment (ioggstream) [inline .github/workflows/publish.yml]:
```suggestion
        uses: actions/checkout@v3
```

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          echo "##[set-output name=imageName;]${IMAGE}:${TAG}"
+      - uses: convictional/trigger-workflow-and-wait@v1.3.0
+        with:
+          owner: teamdigitale
+          repo: dati-semantic-kubernetes
+          github_token: ${{ secrets.DATI_SEMANTIC_KUBERNETES_DEPLOY }}
+          workflow_file_name: update-config.yaml
+          ref: dev
+          wait_interval: 20
+          inputs: '{"serviceName": "dati-semantic-WebVOWL", "imageWithNewTag": "${{ steps.extractImage.outputs.imageName }}"}'
```
Comment (ioggstream) [inline .github/workflows/publish.yml]:
```suggestion
          inputs: '{"serviceName": "${{ env.GITHUB_WORKFLOW }}", "imageWithNewTag": "${{ steps.extractImage.outputs.imageName }}"}'
```

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -0,0 +1,136 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+env:
+  GITHUB_WORKFLOW: dati-semantic-WebVOWL
```
Comment (ioggstream) [inline .github/workflows/publish.yml]:
```suggestion
  GITHUB_WORKFLOW: dati-semantic-webvowl
```

File: Dockerfile [hunk 4]
```diff
@@ -12,6 +12,8 @@ COPY --from=builde_fe ./deploy ../webVowl/deploy
 RUN ./gradlew clean build 
 
 FROM tomcat:9.0.48-jdk11-openjdk-slim
+RUN chown 1001 -R /usr/local/tomcat
```
Comment (ioggstream) [inline Dockerfile:15]: @nnzJd bravo!

File: owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java [hunk 5]
```diff
@@ -204,10 +208,18 @@ private void preParsing(OWLOntology ontology, VowlData vowlData, OWLOntologyMana
...
 				// we dont care
 				// this is a declaration axiom and not an assertion axiom , we are interessted
 				// in assertion to baseConstructors
+=======
+			} catch (RuntimeException e) {
+				// we dont care
+				// this is a declaration axiom and not an assertion axiom , we are interessted
+				// in assertion to baseConstructors
+				logger.warn(e);
+>>>>>>> master
```
Comment (ioggstream) [inline owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java]: @nnzJd Questo è rotto.

**Review by ioggstream — CHANGES_REQUESTED** [review]
Broken files around.

File: owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java [hunk 6]
```diff
@@ -321,6 +337,10 @@ private void processAnnotationProperties(OWLOntology ontology, VowlData vowlData
 					vowlData.addBaseConstructorAnnotation(subject, property, value);
 				} catch (Exception e) {
 					// not interesting
```
Comment (ioggstream) [inline owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java:324]: @nnzJd broken file.

File: owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/server/Owl2VowlController.java [hunk 7]
```diff
@@ -43,8 +43,8 @@ public class Owl2VowlController {
 	private static final String DIRECT_INPUT = "/directInput";
 	private static final String GIZMO_INPUT = "/gizMOInput";
 	
-	public  static       String loadingStatusMsg="";
-	private static Boolean needConversionID=true;
+	public  		     String loadingStatusMsg="";
+	private 			 Boolean needConversionID=true;
```
Comment (ioggstream) [inline owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/server/Owl2VowlController.java]: @nnzJd do we format code before commit?

File: owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java [hunk 8]
```diff
@@ -297,7 +298,7 @@ private void processClasses(OWLOntology ontology, VowlData vowlData) {
 				OwlClassAxiomVisitor temp = new OwlClassAxiomVisitor(vowlData, owlClass);
 				try {
 					owlClassAxiom.accept(temp);
-					temp.Destrucotre();
+					temp.destrucotre();
```
Comment (ioggstream) [inline owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java:301]: @nnzJd ???

File: owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java [hunk 9]
```diff
@@ -204,10 +204,11 @@ private void preParsing(OWLOntology ontology, VowlData vowlData, OWLOntologyMana
 
 				}
 				vowlData.addBaseConstructorAnnotation(subject, property, value);
-			} catch (Exception e) {
+			} catch (RuntimeException e) {
```
Comment (ioggstream) [inline owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java:207]: +1

File: owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java (hunk 8 above)
Comment (nnzJd) [reply owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java:301]: resolve a spotbugs warning

File: owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java (hunk 9 above)
Comment (nnzJd) [reply owl2vowl/src/main/java/it/gov/innovazione/owl2vowl/converter/AbstractConverter.java:207]: resolve a spotbugs warning

**Review by ioggstream — APPROVED** [review]
LGTM

**Pull request merged**